### ISSUE TITLE
Fee reimbursement for trade 02645125

### ISSUE THREAD

**Issue by cbeams** [bisq-network/support#139]
![image](https://user-images.githubusercontent.com/301810/46244237-283b5a80-c3dc-11e8-956d-e70a73d4a086.png)

Maker: ad9f280a88e0e3fe0c9394ed37d6d5e1c94f6393914bfcb137847d61e728a4df
Taker: c360ff3e0df5fbf91a1a9232c671ef1b03692fa614a0fa3326865210385636b2
Deposit: 6d59fb1cb9bc59e82694b5f0f1c405e9fc09f8579927c09d1819410ea6179c09

Buyer's maker fee transaction was not broadcast. Seller's lost taker fees will be reimbursed in #101.

**Comment by KanoczTomas** [comment]
@cbeams could you please assist here. The deposit transaction seems valid to me, when I track its inputs it references the maker/taker txids. I might be looking at it wrong though.

**Comment by cbeams** [comment]
Good catch, @KanoczTomas. It looks like I incorrectly cancelled this trade as failed. In any case, we should still go ahead with reimbursing both maker and taker transactions given that this was my mistake and that the trade actually could have been completed.
